arch/arm/soc/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -107,6 +108,7 @@ config SOC_NRF52832
 
 config SOC_NRF52840
 	depends on SOC_SERIES_NRF52X
+	select CPU_HAS_FPU
```

Comment:
This addition is no longer needed (see 2 lines below).

Reply:
thanks